messages/ulog/baro.yaml

```diff
@@ -0,0 +1,9 @@
+name: baro
+description: Contains barometer temperature, pressure and altitude
+fields:
+  - name: temp
+    type: float
+  - name: press
```

Comment:
Please either add units in the comment, or in the variable name (`press_bar`).

Reply:
Additionally, there is no reason to abbreviate these IMO. `pressure` and `temperature` are short enough.

Reply:
Thanks!